conf/messages

```diff
@@ -19,22 +19,34 @@ error.maxLength = You must not enter more than {0} figures
 error.email= Enter a valid email address, like yourname@example.com
 
 ##Errors
+error.email.invalidchars=Business email address must only contain letters a to z, numbers, full stops, hyphens, underscores and @ signs
```

Comment:
The DES subscription API doesn't allow @ signs from the looks of its regex `^[A-Za-z0-9 \-,.&'\\/]{1,40}$`